editor/plugins/polygon_2d_editor_plugin.h

```diff
@@ -50,50 +50,60 @@ class VScrollBar;
 class Polygon2DEditor : public AbstractPolygon2DEditor {
 	GDCLASS(Polygon2DEditor, AbstractPolygon2DEditor);
 
-	enum Mode {
-		MODE_EDIT_UV = MODE_CONT,
```

Comment:
This enum is some kind of continuation from AbstractPolygon2DEditor, but the usage is totally unrelated (especially after removing the dialog opening "uv" button), so I changed the code to reflect that.